src/JSONSchemasInterface.js

```diff
@@ -0,0 +1,61 @@
+import { schemas } from "@exabyte-io/esse.js/schemas";
+import mergeAllOf from "json-schema-merge-allof";
+
+const schemaCache = new Map();
+
+export const JSONSchemasInterface = {
+    schemas() {
+        return schemas;
```

Comment:
1. This could be a static variable
2. Maybe `allSchemas` for clarity
3. Is this ever used?

Reply:
Let's remove